src/orders/components/OrderDraftDetailsSummary/OrderDraftDetailsSummary.tsx

```diff
@@ -47,9 +47,9 @@ const useStyles = makeStyles(
 );
 
 const messages = defineMessages({
-  addCustomerInfo: {
-    defaultMessage: "add customer first",
-    description: "add customer first label"
+  addShippingAddressInfo: {
```

Comment:
Since we decided to use local messages files for messages it'd be a good moment to move it? :)